webapp/.eslintrc.json

```diff
@@ -209,7 +209,7 @@
 		"react/jsx-uses-react": 2,
 		"react/jsx-uses-vars": 2,
 		"react/no-danger": 0,
-		"react/no-did-mount-set-state": 2,
+		"react/no-did-mount-set-state": [2, "allow-in-func"],
```

Comment:
Yep, I'll do that. I was trying to avoid adding `const self = this;` everywhere, but I think any of the places that we're do that now could avoid it